content/epmdlessless.md

```diff
@@ -0,0 +1,124 @@
++++
+author = "Tristan Sloughter"
+type="post"
+categories = ["Erlang"]
+date = 2020-11-30T10:41:00Z
+description = ""
+draft = false
+slug = "epmdlessless"
+tags = ["Erlang"]
+title = "Running Erlang Releases without EPMD on OTP 23.1+"
++++
+
+With Erlang/OTP 23.1 and relx 4.2 (included in rebar3 3.14.3) it is now possible
+to conveniently run Erlang releases without relying on EPMD or any third party
+dependency, like [epmdless](https://github.com/tsloughter/epmdless), for opening a
+remote shell or clustering nodes.
+
+The Erlang Port Mapper Daemon (EPMD) is a separate process that is started, if
```

Comment:
```suggestion
EPMD is a separate process that is started, if
```
moved the acronym definition in my previous suggestion; if accepted, it has become redundant here.